File: Assignment32/program5.java
import java.lang.*;
import java.util.*;

class Bitwise
{
    public int ToggleBit(int iNo)
    {

         int iMask = 0X10000001;
         int iResult = iMask ^ iNo;
         
    return iResult;
    }
}


public class program5{

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the number");
        int value = sobj.nextInt();

        Bitwise obj = new Bitwise();
        int iret = obj.ToggleBit(value);

        
        System.out.println("updated number "+iret);
    }
    
}

File: Assignment39/program5.c
    #include<stdio.h>
void Reverse(int ino)
{
    while(ino!=0)
    {   
        
        printf("%d",ino%10);
        
        ino=ino/10;
    }
}

void ReverseR(int ino)
{
    if(ino!=0)
    {   
        
        printf("%d",ino%10);
        
        ino=ino/10;
        ReverseR(ino);
    }
}
int main()
{
    int iRet=0;
    int value=0;
    printf("Enter number: \n");
    scanf("%d",&value);
    ReverseR(value);
}

